Update outlook-prompt-password-modern-authentication-enabled.md (#1725)
MicrosoftDocs/OfficeDocs-Support: Outlook/Client/authentication/outlook-prompt-password-modern-authentication-enabled.md

> [!IMPORTANT]
> Follow the steps in this section carefully. Serious problems might occur if you modify the registry incorrectly. Before you modify it, [back up the registry for restoration](https://support.microsoft.com/help/322756) in case problems occur.

> Create the following registry key to force Outlook to use the newer authentication method for web services, such as EWS and Autodiscover. It is recommended that users force Outlook to use Modern Authentication.

Reviewer: ```suggestion
Create the following registry key to force Outlook to use a newer authentication method for web services, such as EWS and Autodiscover. We recommend that users force Outlook to use Modern Authentication.
```
